packages/create-app/templates/default-app/.dockerignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
```

Comment:
@lowjoel Using a dockerignore instead of changing context. Containers start fairly quick on 2018 15'in MBP

Reply:
Yeah. That's fine. I had both in my source repository because the repository contains both the code, as well as the deployment/infrastructure definitions.